CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.22.1
```

Comment:
This needs to be changed to ##1.23.0-dev, since we are going to bump the major version this time.